src/main/resources/static/index.html

```diff
@@ -16,16 +16,18 @@
         <!--signup will be a button next to login that turns into a signout after a user is logged in-->
 
         <div class="ui right floated header">
+            <button class="ui left floated green inverted button" id="manageUsersNavButton" style="display: none;"
```

Comment:
Could you possibly change the color of the button so that it doesn't blend in with the background.

Reply:
Done deal. I have no idea what happened to the button in my initial pull request.. I must have lost it when I did a last pull and merge from master before I pull requested. It's back.